CoffeeChat/Controller/OnboardingInterestsViewController.swift

```diff
@@ -50,18 +54,21 @@ class OnboardingInterestsViewController: UIViewController {
 
     override func viewDidLoad() {
         super.viewDidLoad()
-        view.backgroundColor = .backgroundWhite
+        view.backgroundColor = .backgroundLightGreen
+        navigationController?.navigationBar.isHidden = true
 
-        titleLabel.text = "What are your interests?"
-        titleLabel.font = .systemFont(ofSize: 24, weight: .bold)
+        titleLabel.text = "What do you love?"
+        titleLabel.font = ._24CircularStdMedium
         view.addSubview(titleLabel)
 
         tableView.delegate = self
         tableView.dataSource = self
         tableView.register(OnboardingTableViewCell.self, forCellReuseIdentifier: OnboardingTableViewCell.reuseIdentifier)
         tableView.isScrollEnabled = true
         tableView.clipsToBounds = true
+        tableView.backgroundColor = .backgroundLightGreen
         tableView.allowsMultipleSelection = true
+//        tableView.bounces = false
```

Comment:
remove

Reply:
@pnor I commented this out because I wasn't sure if we want the table to bounce in the interests view controller. Do you have an opinion on this?

Reply:
I'm good with removing it, I didn't realize it was bouncing before. We can ask design though for the final say